src/shipping/consignment-action-creator.ts

```diff
@@ -150,7 +151,7 @@ export default class ConsignmentActionCreator {
 
     updateAddress(
         address: AddressRequestBody,
-        options?: RequestOptions
+        options?: RequestOptions | UpdateShippingAddressRequestOptions
```

Comment:
I think this could just be `UpdateShippingAddressRequestOptions` as it is a subset of `RequestOptions`